lib/orderbook/OrderBookRepository.ts

```diff
@@ -88,17 +90,8 @@ class OrderbookRepository {
     return this.models.Order.bulkCreate(<db.OrderAttributes[]>orders);
   }
 
-  public updateOrder = async (orders: orders.StampedOrder, quantityLeft: number) => {
-    await this.models.Order.update({ quantity: quantityLeft }, { where: { id: orders.id } });
-  }
-
-  public getOrderQuantity = async (order: orders.StampedOrder): Promise<number> => {
-    const result = await this.models.Order.find({
-      where: { id: order.id },
-      raw: true,
-      attributes: ['quantity'],
-    }) as db.OrderInstance;
-    return Number(result.quantity);
+  public updateOrderQuantity = async (orderId: string, usedQuantity: number) => {
+    await this.models.Order.update({ quantity: this.sequelize.literal(`quantity - ${usedQuantity}`) }, { where: { id: orderId } });
```

Comment:
i'm still getting a type warning here, despite the change you made on `types/db.ts`. see if there's a solution for that.

Reply:
I don't get a warning at all. Do you get it when compiling or linting?